generator/explores/glean_ping_explore.py

```diff
@@ -10,9 +15,28 @@ class GleanPingExplore(PingExplore):
 
     type: str = "glean_ping_explore"
 
+    def _to_lookml(self, v1_name: Optional[str]) -> dict:
+        """Generate LookML to represent this explore."""
+        repo = next((r for r in GleanPing.get_repos() if r["name"] == v1_name))
+        glean_app = GleanPing(repo)
+        ping_description = glean_app.get_ping_descriptions()[self.name]
+        return {
+            **super()._to_lookml(v1_name),
+            **{"description": f"Explore for the {self.name} ping. {ping_description}"},
+        }
```

Comment:
I know this works but I think it may be extra confusing to look at. How do you feel about something like, just for clarity:
```suggestion
        lookml = super()._to_lookml(v1_name)
        lookml["description"] = f"Explore for the {self.name} ping. {ping_description}"
        return lookml
```

Reply:
Yup. Python 3.9 will make this easier to do elegantly.